charts/argo-cd/templates/argocd-applicationset/clusterrole.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.createClusterRoles }}
```

Comment:
Not all users use `ApplicationSet in any namespace` feature, so I think it's better to add a new flag. 🤔

Reply:
I think so.
However, to enable the application in any namespace feature, we must set .Values.createClusterRoles=true or .Values.controller.clusterAdminAccess=true ( .controller.clusterAdminAccess is deprecated). .
Therefore, we believe it is good to be able to enable application set in any namespace feature in the same way (since those who use application in any namespace are also likely to use application set in any namespace).

Reply:
I submitted a PR to argo-cd repo (https://github.com/argoproj/argo-cd/pull/16699).
After the above PR is merged, it would be better to merge this PR.

Reply:
Thank you !
Let us add `await-upstreaming` label at this PR.

Reply:
https://github.com/argoproj/argo-cd/pull/16699 is merged!

Please re-review.
@yu-croco

Reply:
But not released.

Reply:
Ah @tico24 is right, merging doesn't mean they release new version. 😇

Reply:
New Argo CD version is released!!!
Could you re-review this pr?
@yu-croco

Reply:
@mugioka 
v2.9.5 (The latest version) doesn't contain the diff. Which version are you pointing out ..?
Ref: https://github.com/argoproj/argo-cd/tree/v2.9.5/manifests/cluster-rbac

Reply:
Oh, I'm sorry, I totally thought it had been released.
I will request re-review after I confirm that it has been released.
@yu-croco

Reply:
@yu-croco We can probably review this - 2.10.0 is released and this change will be cherry-picked by upstream for next release patch release.

Reply:
Can we add an and expression with a new value (eg. `.Values.applicationSet.allowAnyNamespace` or `.Values.features.applicationSetInAnyNamespace`)

This feature needs to be disabled by default IMHO. We at Swiss Post don't want to allow this and also this feature is beta.
Ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Appset-Any-Namespace/

Reply:
OK
> Can we add an and expression with a new value (eg. .Values.applicationSet.allowAnyNamespace or .Values.features.applicationSetInAnyNamespace)

i will add it.

Reply:
@yu-croco 
Sorry for delay,,,

I fixed feedback.
Please re-review this pr.